db/migrate/20250814153241_create_mhv_metrics_unique_user_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMHVMetricsUniqueUserEvents < ActiveRecord::Migration[7.2]
+  def change
+    create_table :mhv_metrics_unique_user_events, if_not_exists: true, id: false do |t|
+      # Note: user_id intentionally has NO foreign key constraint by design
+      # Benefits: Performance optimization for high-volume inserts, historical data preservation
+      # even if users are deleted, and operational simplicity for this analytics table
+      t.uuid :user_id, null: false, comment: 'Unique user identifier'
+      t.string :event_name, limit: 50, null: false, comment: 'Event type name'
+      t.timestamp :created_at, null: false, default: -> { 'CURRENT_TIMESTAMP' }, comment: 'Auto-populated creation timestamp'
```

Comment:
It's enough to say `t.timestamp :created_at` on this line. The default operation is to populate this with a timestamp when a record is saved.